enumeration_experimental.go

```diff
@@ -385,6 +386,32 @@ func (a *ArraySchema) AddEnumeration(e *Enumeration) error {
 	return nil
 }
 
+// EnumerationFromName gets an Enumeration from the ArraySchema by name
+func (a *ArraySchema) EnumerationFromName(name string) (*Enumeration, error) {
+	enum := &Enumeration{context: a.context}
+	cName := C.CString(name)
+	defer C.free(unsafe.Pointer(cName))
+	ret := C.tiledb_array_schema_get_enumeration_from_name(a.context.tiledbContext, a.tiledbArraySchema, cName, &enum.tiledbEnum)
+	if ret != C.TILEDB_OK {
+		return nil, fmt.Errorf("Error getting enumeration from name: %s", a.context.LastError())
```

Comment:
Nit: Errors should be lower case (see [Go's documentation](https://go.dev/wiki/CodeReviewComments#error-strings)) and we should use `%w` for error wrapping, which allows the caller to use `errors.{Unwrap,Is,As}`.
```suggestion
		return nil, fmt.Errorf("error getting enumeration from name: %w", a.context.LastError())
```